### DISCUSSION TITLE
Increase the batch size query for GraphQL (currently limited to 30 items)

### DISCUSSION THREAD

**Discussion by MCYouks** [supabase/supabase#17727] [Feature Requests]
# Current state

At the moment, the size of query batch is limited to 30 items per Graphql query, against 1000 items using the Supabase SDK.

# Problem

This batch size being very low, we need to create auto pagination utilities to query more than 30 items at once.

# Proposed solution

Increase the batch size for GraphQL queries. 1000 items would be ideal.

**Comment by fredro84** [comment]
Stumbled upon your request, but found in the documentation that you can set it yourself as a comment
https://supabase.com/docs/guides/graphql/configuration#max-rows

comment on schema public is e'@graphql({"max_rows": 100})';

So guess you can close it.
